pt/lessons/basics/strings.md

```diff
@@ -1,6 +1,8 @@
 ---
-version: 0.9.1
+version: 1.1.1
 title: Strings
+redirect_from:
```

Comment:
Thanks, @brunoliveira8!
The `redirect_from` clause [should not be used](https://github.com/elixirschool/elixirschool/blob/master/README.md#translated-lesson) in translations and new lessons.